comparisonScaling.cpp

```diff
@@ -41,10 +43,13 @@ int main(int argc, char** argv) {
         {RustFmphGoContender(N, 1.5).run();}
         {RustFmphContender(N, 1.5).run();}
         {BBHashContender(N, 1.5, 0).run();}
+        {MorphisHashContender<64,5>(N, 2000).run();}
         {BipartiteShockHashContender<64>(N, 2000).run();}
         if (Contender::numThreads == 1) {
             {RecSplitContender<14>(N, 2000).run();}
+            {MorphisHashContender<40,5>(N, 2000).run();}
             {ShockHashContender<40>(N, 2000).run();}
+            {MorphisHashFlatContender<40,5>(N).run();}
```

Comment:
This plot otherwise has only one copy of each competitor. Was the duplication intended?